contracts/BaseStrategy.sol

```diff
@@ -149,6 +153,21 @@ abstract contract BaseStrategy {
      */
     function name() external virtual pure returns (string memory);
 
+    /**
+     * @notice
+     *  The amount (priced in want) of the total assets managed by this strategy should not count
+     *  towards Yearn's TVL calculations.
+     * @dev
+     *  You can override this field to set it to a non-zero value if some of the assets of this
+     *  Strategy is somehow delegated inside another part of of Yearn's ecosystem e.g. another Vault.
+     *  Note that this value must be strictly less than or equal to the amount provided by
+     *  `estimatedTotalAssets()` below, as the TVL calc will be total assets minus delegated assets.
+     * @return True if positions should be included in Yearn's Total Value Locked (TVL) calculation.
```

Comment:
Is this comment correct? Return value is uint

Reply:
ah yeah I think that's probably leftover from when this was originally going to be boolean

Reply:
crap, missed this